game-core/src/main/java/games/strategy/net/Node.java

```diff
@@ -12,25 +8,22 @@
 import lombok.AllArgsConstructor;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
-import lombok.NoArgsConstructor;
 import lombok.ToString;
 
 /**
  * Written very often over the network, so make externalizable to make faster and reduce traffic.
  */
 @ToString
 @Getter(onMethod_ = {@Override})
-// NoArgsConstructor needed to support Externalizable
-@NoArgsConstructor
 @AllArgsConstructor
 @EqualsAndHashCode(exclude = "name")
```

Comment:
Doesn't seem to work. Seems like `@AllArgsConstructor` is only being added if no constructor exists yet, which is not the case.
At this point this we'd just trade an annotation for a different one with now additional value.

Reply:
You should be able to replace `@AllArgsConstructor` and `@ToString` with `@Value`

Normally @Value would be the same as all 4 of the annotations listed, toString, equals, allArgs, and getter.

Because there is the exclude and onMethod, perhaps it's better to have the individual annotations.